app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def create_dirs(path_to_create: str) -> None:
+    try:
+        os.makedirs(os.path.dirname(path_to_create), exist_ok=True)
+    except OSError as e:
+        print(f"Error creating directories: {e}")
+
+
+def copy_and_delete_file(file_from: str, file_to: str) -> None:
+    try:
+        with open(file_from, "r") as read_file, \
+                open(file_to, "x") as write_file:
```

Comment:
we don't use \ as a line breaker. read this message please
https://mate.academy/chats/244801?selectedMessageId=3129519